packages/tailwindcss-language-server/src/oxide.ts

```diff
@@ -58,10 +58,32 @@ declare namespace OxideV3 {
   }
 }
 
+// This covers the Oxide API from v4.1.0+
+declare namespace OxideV5 {
+  interface GlobEntry {
+    base: string
+    pattern: string
+    negated: boolean
+  }
+
+  interface ScannerOptions {
+    sources: Array<GlobEntry>
```

Comment:
Not strictly necessary, but this matches the actual API.
```suggestion
    sources?: Array<SourceEntry>
```